<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parking_Systen</class>
 <widget class="QMainWindow" name="Parking_Systen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>주차정산시스템</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="into_parking_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>540</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>입차</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rate_check_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>요금조회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_parking_btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>440</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>주차 현황</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leave_parking_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>440</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>출차</string>
    </property>
   </widget>
   <widget class="QPushButton" name="end_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>540</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Test">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>540</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>차량번호</string>
    </property>
   </widget>
   <widget class="QLabel" name="view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>540</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputText">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
